src/sentry/integrations/discord/webhooks/command.py

```diff
@@ -37,25 +41,35 @@ class DiscordCommandHandler(DiscordInteractionHandler):
 
     def handle(self) -> Response:
         command_name = self.request.get_command_name()
```

Comment:
nit: should we update `get_command_name` to check the lowercase version of the command? i believe we are already doing it for slack and MSTeams

Reply:
Actually, the check is always done case-insensitively (thanks to the `.casefold()` calls in the `CommandSlug` methods), so that shouldn't be necessary.

The downcasing on MSTeams is actually unnecessary, and I suppose maybe we shouldn't do that in case we ever want to preserve case for non-slug arguments in the future. So I took it out (d5374d72e4b74e745472833118aae2cfa7f64bba) instead. Thanks for the catch.